torch/__init__.py

```diff
@@ -410,6 +411,22 @@ def is_deterministic():
     """
     return _C._get_deterministic()
 
+def set_warnOnlyOnce(b):
+    r"""Convert the TORCH_WARN_ONCE C++ macro into TORCH_WARN for tests, so
```

Comment:
Let's write this documentation to be user-facing, so no mention of the macro.

When this flag is True (default) then some PyTorch warnings may only appear once per process. This helps avoid excessive warning information. Setting it to False causes these warnings to always appear, which may be helpful when debugging.

Reply:
done